test-environment.md

```diff
@@ -100,6 +103,31 @@ Other differences between Production and MT:
 - We allow 10,000 incorrect PIN attempts before locking the Vipps user's account
 - There are no settlement reports and no SFTP service.
 
+## Access to the test environment
+
+### Merchants
+
+All merchants with API access have access to the Vipps test environment.
+A test sale unit is automatically created.
```

Comment:
Maybe add a bit more to the last sentence there:
Something like this perhaps? 
"A test sale unit is automatically created, and the details can be found on the developer page in https://portal.vipps.no/"

Reply:
Good idea!